### ISSUE TITLE
Published version 3.4.0 does not work

### ISSUE THREAD

**Issue by kammoh** [vagmcs/Optimus#40]
Running any of the examples (e.g. https://github.com/vagmcs/Optimus/blob/master/docs/mixed_integer.md) results in an exception:

```
Exception in thread "main" java.io.FileNotFoundException: /home/vagmcs/Usr/work/dev/Optimus/core/target/scala-2.13/scoverage-data/scoverage.measurements.1 (No such file or directory)
	at java.base/java.io.FileOutputStream.open0(Native Method)
	at java.base/java.io.FileOutputStream.open(FileOutputStream.java:298)
	at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:237)
	at java.base/java.io.FileWriter.<init>(FileWriter.java:113)
	at scoverage.Invoker$.$anonfun$invoked$1(Invoker.scala:55)
Invoker.scala:55
	at scala.collection.mutable.HashMap.getOrElseUpdate(HashMap.scala:454)
HashMap.scala:454
	at scoverage.Invoker$.invoked(Invoker.scala:55)
Invoker.scala:55
	at optimus.optimization.MPModel.<init>(MPModel.scala:36)
MPModel.scala:36
```

`localPublish`ed version with `coverageEnabled := false` works as expected.

Seems to be related to:
https://github.com/scoverage/sbt-scoverage/issues/84 and https://github.com/scoverage/sbt-scoverage/issues/306

**Comment by vagmcs** [comment]
Hello @kammoh,

What version of Java and Scala do you use?

**Comment by kammoh** [comment]
Scala 2.13.5, OpenJDK 11

**Comment by vagmcs** [comment]
Hi @kammoh,

There is an import missing from the MIP example in the documentation. You have to add,

```scala
import optimus.optimization.model.MPIntVar
```

Apart from that it works fine for me. If this is not the problem in your case you have to provide me with more details. Maybe the code you are trying to run and the way you run it, e.g., from some IDE or SBT console.

**Comment by kammoh** [comment]
Hi @vagmcs ,
Please have a look at the reported error message and the referenced `sbt-coverage` issues. This has nothing to do with the missing import!
You can see the error on Scastie:
https://scastie.scala-lang.org/hnymyJJ4SNqnrfpRT5Zy2Q

Upgrading `sbt-coverage` could be a thing to try, but as it seems not to be resolved upstream, you need to disable `sbt-coverate` during publish (`coverageEnabled := false`).

**Comment by kammoh** [comment]
Maybe the reason you're not observing this issue on your local machine is that your build-time coverage file (`/home/vagmcs/Usr/work/dev/Optimus/core/target/scala-2.13/scoverage-data/scoverage.measurements.4463`) actually exists there?

**Comment by vagmcs** [comment]
Hello @kammoh,

It seems there is indeed a problem with the coverage option. I released a patched version 3.4.1. Can you please try it out?

**Comment by kammoh** [comment]
Yes, it solves the issue! Thank you for the quick fix!
